spec/lib/vanagon/platform_spec.rb

```diff
@@ -4,22 +4,31 @@
   let(:platforms) do
     [
       {
-        :name         => "debian-6-i386",
-        :os_name      => "debian",
-        :os_version   => "6",
-        :architecture => "i386",
+        :name                   => "debian-6-i386",
+        :os_name                => "debian",
+        :os_version             => "6",
+        :architecture           => "i386",
+        :output_dir             => "debian/6/i386",
+        :output_dir_with_target => "debian/6/thing/i386",
```

Comment:
So to actually answer the question, no we will not change anything in packaging, we will just be adding to the platform configs in the project.